docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -75,7 +75,7 @@ The Astra API endpoint is available in the Astra Portal (e.g. `https://<ASTRA_DB
 +
 Create an OpenAI key at https://platform.openai.com/ (e.g. `sk-xxxx`).
 +
-You must have an existing collection in Astra (e.g. `test`). If you don't have one, create one in the Astra portal, or with the Data API (using the `.env` values from the next step):
+You must have an existing collection in {db-vector} (e.g. `test`). If you don't have one, create one in the {astra-ui}, or with the Data API (using the `.env` values from the next step):
```

Comment:
Per the Google Developer Doc Style Guide, avoid using "e.g". Use "for example" instead.